src/components/UnreadActionIndicator.js

```diff
@@ -1,29 +1,17 @@
 import React from 'react';
-import {Animated, View} from 'react-native';
-import PropTypes from 'prop-types';
+import {View} from 'react-native';
```

Comment:
Any particular reason you decided to nix the animation? NAB, just curious

Reply:
Animations can make a lot of extra logic for a questionable ROI, so if you don't want to implement them I think it's fine  to focus on getting the proper functionality bug-free, and then if we want we can circle back and have a contributor animate it.

Reply:
Good point. We did already have a contributor implement it - so if this is a possibility...

>we can circle back and have a contributor animate it

Maybe we should just leave the animation in for now?

Reply:
Well, I got rid of it mainly because we are no longer hiding the marker in front of the user, so the need for an animation felt mostly gone 🤷

Reply:
Okay, makes sense.